src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
+
+  return (
+    <header style={headerStyle}>
```

Comment:
Very smart, did you get this from the tutorial?

I want to use this myself, it looks lazy and I love it!

Reply:
Yeap. Part 3 of the tutorial. There you'll find all the methods